server/events.go

```diff
@@ -138,14 +138,19 @@ const DisconnectEventMsgType = "io.nats.server.advisory.v1.client_disconnect"
 // updates in the absence of any changes.
 type AccountNumConns struct {
 	TypedEvent
-	Server        ServerInfo `json:"server"`
-	Account       string     `json:"acc"`
-	Conns         int        `json:"conns"`
-	LeafNodes     int        `json:"leafnodes"`
-	TotalConns    int        `json:"total_conns"`
-	Sent          DataStats  `json:"sent"`
-	Received      DataStats  `json:"received"`
-	SlowConsumers int64      `json:"slow_consumers"`
+	Server ServerInfo `json:"server"`
+	*AccountStat
```

Comment:
Why `*AccountStat` and not `AccountStat`? Less change of breaking existing apps relying on fields to always be present. I see that accConns() always return &AccountStat{} so it is not like there are cases where the returned object would be smaller.

Reply:
fixed

Reply:
Changed it to AccountStat. But kept it `*` in other occurances as `[]*` is used in monitoring a lot.
Example in connz
```
// Connz represents detailed information on current client connections.
type Connz struct {
	ID       string      `json:"server_id"`
	Now      time.Time   `json:"now"`
	NumConns int         `json:"num_connections"`
	Total    int         `json:"total"`
	Offset   int         `json:"offset"`
	Limit    int         `json:"limit"`
	Conns    []*ConnInfo `json:"connections"`
}

```

Reply:
My comment here was because AccountNumConns{} already existed and so changing from the fields to embedding `*AccountStat` could cause existing apps to crash if one day `*AccountStat` is nil. Had it been never released, I would not even have commented.